Skimming/higgsTauTau/kSkimming_run2_cfg.py

```diff
@@ -541,12 +539,11 @@ def getBaseConfig( globaltag= 'START70_V7::All',
 
 	process.p *= process.nEventsFiltered
 	process.p *= process.nNegEventsFiltered
-	process.kappaTuple.active += cms.vstring('FilterSummary')
 
 	## ------------------------------------------------------------------------
 
 	## if needed adapt output filename
-	process.p *= process.kappaOut
+	process.ep *= process.kappaOut
```

Comment:
Just for my curiosity: what is the difference between `process.p` and `process.ep`?